handbook/brand.md

```diff
@@ -33,14 +33,14 @@ Another example to consider is industry jargon that may now be inappropriate. Wh
 Writing at Fleet shares the same principles as [Communicating as Fleet.](https://fleetdm.com/handbook/brand#communicating-as-fleet) Every piece of content we write should be consistent with our company and brand values. To help succeed, we encourage our writers to apply a design thinking approach to their writing by following these principles:
 
 - **Empathize** - Who is the reader? Why will they read it? What do they hope to get from it?
-- **Define** - What are you writing about? What action do you want from the reader?
+- **Define** - What is the subject of your writing? What action do you want from the reader?
```

Comment:
```suggestion
- **Define** - What is the subject? What action do you want from the reader?
```
Yeah, good. Adding "subject" here is better. Let's shorten it because we've already established in the paragraph above that writing is the subject of these principles 👍🏻